### ISSUE TITLE
Communication Devices

### ISSUE THREAD

**Issue by Boy0000** [MineInAbyss/server-config#96]
### Item Type

Tool

### Item name

Communication Devices

### Item Info

They are used to maintain contact with each other while they are too far away from each other to speak directly. They have the shape of a small round speaker that is inserted directly into a Delver's ear, similar to real life earplugs.

The device presumably has a microphone capable of picking up a person's voice, and a receiver and transmitter capable of broadcasting signals to other similar devices within range.

Communication Devices have a limited effective range. The sound quality will diminish the further apart two devices are from each other until it either becomes unintelligible or the devices disconnect entirely.

They are also subject to disruption from the forcefield, which is especially troublesome if they are used to communicate between layers, although they can still function if the units are relatively close to one another. It is unknown if they can be chain wired to provide a wider range.

### Item Model/Texture

[this](https://madeinabyss.fandom.com/wiki/Communication_Device)

### Obtainability

Craftable via recipe / Vendors in Orth

### Related recipe ideas

No specific recipe yet

### Additional info

Basically what we do with [proximity voice](https://www.curseforge.com/minecraft/bukkit-plugins/simple-voice-chat), but as easy to use as possible. If by some miracle we don't need a mod well. Otherwise bind something of the sort to only a specific item?

### Agreements

- [X] I have searched for and ensured there isn't already an open suggestion similar to this.
- [X] I have ensured the relic I'm suggesting isn't already on the server.
